contrib/testnet/Dockerfiles/Dockerfile.arch

```diff
@@ -28,12 +28,16 @@
 
 FROM greyltc/archlinux:latest
 
+ARG userid
+
+ENV userid=$userid
```

Comment:
```
Build Web Docker image? [httpd:2.4] [Y/n] y                                                                                                                                                                       
Sending build context to Docker daemon    741MB                                                                                                                                                     
Step 1/1 : FROM httpd:2.4                                                                                                                                                                                         
 ---> fb2f3851a971                                                                                                                                                                                                
[Warning] One or more build-args [userid] were not consumed
```

I'm imagining this is expected because the apache dockerfile wasn't updated (and it doesn't need to be)?

Reply:
No, I missed this when refactoring. Will remove the build-arg from the web Docker image.